src/export.sh

```diff
@@ -36,7 +36,7 @@ export_project() {
 export_projects() {
     declare month="${1:-$(date -v '-25d' +"%Y-%m")}"
 
-    if [[ ! "$month" =~ ^201[0-9]-[0-9]{2}$ ]]; then
+    if [[ ! "$month" =~ ^20[0-9]{2}-[0-9]{2}$ ]]; then
```

Comment:
lol :) mam nadzieje że to nie ja pisałem :)

Reply:
wyjebie się w 2100 :)

Reply:
I planned this sabotage from the start :>